src/pages/Lend/AlgoliaPagination.vue

```diff
@@ -1,103 +1,155 @@
 <template>
-	<ais-pagination :padding="padding">
-		<ul class="ais-Pagination-list"
-			slot-scope="{
-			currentRefinement,
-			nbPages,
-			pages,
-			isFirstPage,
-			isLastPage,
-			refine,
-			createURL
-			}"
+	<ul class="ais-Pagination-list">
+		<!-- previous page -->
+		<li
+			class="ais-Pagination-item ais-Pagination-item--previousPage"
+			:class="{ 'ais-Pagination-item--disabled' : isFirstPage }">
+			<span
+				v-if="isFirstPage"
+				aria-label="Previous (Inactive)"
+				class="ais-Pagination-link"
+			>
+				<kv-icon name="triangle" class="disabled" />
+			</span>
+			<a
+				v-if="!isFirstPage"
+				:href="createUrl(currentRefinement - 1)"
+				@click.prevent="refine(currentRefinement - 1)"
+				aria-label="Previous"
+				class="ais-Pagination-link"
+			>
+				<kv-icon name="triangle" />
+			</a>
+		</li>
+
+		<li v-for="(number, index) in numbers"
+			:key="number || -index"
```

Comment:
cc @emuvente would this be an ok change to make?

Reply:
`key` is used to prevent unnecessary re-rendering. What makes each of these elements most unique is their page number, so basing the key on that allows more elements to be re-used between renders. However, not all elements have a number (i.e. ellipsis elements), and so for those we choose a key that won't interfere with other keys, thus `-index`.